### DISCUSSION TITLE
Anyone read the "Teaching and Learning with Jupyter" Book?

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/pythia-foundations#156] [Show and tell]
It looks like there is a book online on how to teach and learn with Jupyter:

https://jupyter4edu.github.io/jupyter-edu-book/

It sounds like it could be an interesting, and possibly useful, read.

**Comment by brian-rose** [comment]
I just took a look. I'd never seen this before. There's some excellent content here for educators. I particularly like Chapter 4, [A catalogue of pedagogical patterns](https://jupyter4edu.github.io/jupyter-edu-book/catalogue.html) which succinctly describes various ways to use notebooks in classrooms and courses.

It looks like the book has not been maintained. Last update was a few years ago, and the build system (which predates the existence of JupyterBook) seems to be broken. Similarly the technical content is somewhat out of date (e.g. no mention of Jupyter lab)

**Comment by brian-rose** [comment]
Source is here: https://github.com/jupyter4edu/jupyter-edu-book

**Comment by kmpaul** [comment]
Yes.  The build system is really outdated, but the content could be useful.  Might also be interesting to see if the authors might be willing to share content (or we could just cite it).

**Comment by ktyle** [comment]
Ahhh, this is one of the publications I discovered when doing initial planning for my class in the summer of 2020! Some well-known names in there, such as Carol Willing and Loren Barba ... I wonder if this arose from the 2019 SciPy conference (I met Kyle Niemeyer at that meeting and discussed educational matters with him)? Most relevant though is Lindsay Heagy ... being a Co-PI with Fernando Perez on the Jupyter Meets the Earth EarthCube project. Perhaps we could reach out to her about adapting the article into our material?

**Reply by kmpaul** [reply]
I am also Co-PI on the Jupyter Meets the Earth project.  😄  I'm sure Lindsey would be supportive, though she's moved on to other things these days.  She might know who to talk to about using the material, though.
